### ISSUE TITLE
Rethinking Batch Normalization in Transformers

### ISSUE THREAD

**Issue by sanxing-chen** [STayinloves/paper-note#16]
Authors: Sheng Shen, Zhewei Yao, Amir Gholami, Michael Mahoney, Kurt Keutzer
Time: 17 Mar 2020
Note:
- It's widely known that BN cannot be naively applied to NLP models
  - NLP data has a very large variance throughout training
- The paper proposes Power Normalization (PN) which modifies BN while preserving good theoretic property.
- Experiments show that PN outperforms BN and LN in MT and LM tasks
- The analysis shows that PN leads to a more well-conditioned embedding layer and is robust to small-batch statistic
